CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ This can also be enabled programmatically with `warnings.simplefilter('default',
 * individual `/Resources` directories are now properly created for each document page. This change ensures better compliance with the PDF specification but results in a slight increase in the size of PDF documents. You can still use the old behavior by setting `FPDF().single_resources_object = True`
 * line size calculation for fragments when text shaping is used
 ### Changed
+* Removed an obscure and undocumented feature of [`FPDF.write_html()`](https://py-pdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.write_html), which magically passed local variables as arguments.
```

Comment:
End users won't know exactly what is is this feature when reading this...

Could you maybe add a link to issue #1198 so that users that want to better understand what changed can quickly get more information about this?

Reply:
done.